<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>base</class>
 <widget class="QWidget" name="base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="channel_top_frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="type_label">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="type_combo">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Intensity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Color</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>White</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Amber</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cyan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Magenta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Purple</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Pan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Tilt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Beam</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gobo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Strobe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nothing</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="channel_table_container">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="channel_table">
        <property name="rowCount">
         <number>9</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Minimum Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Maximum Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
